### ISSUE TITLE
Text-classification dataset creation train/val/test splits

### ISSUE THREAD

**Issue by snu-ceyda** [NVIDIA/DIGITS#1624]
Currently it looks like we can only specify csv files for train and validation sets.
https://github.com/NVIDIA/DIGITS/blob/master/examples/text-classification/create-dataset.png
It would be really useful if we could also specify a test set (csv) 
I know it is possible to use a csv file for testing ([like this](https://github.com/NVIDIA/DIGITS/blob/master/examples/text-classification/inference-form.png)) but when it is a biggish(~7000 examples) file it doesn't work well or at all
